### ISSUE TITLE
Monitor function

### ISSUE THREAD

**Issue by BrianWhiteaker** [BrianWhiteaker/HMC#7]
We need a monitor function that is computationally fast and is not implicit.  
Question: Could we use the osculating circle?

**Comment by BrianWhiteaker** [comment]
Also, if we can find some bounding value $b$ on $\kappa$ beyond which standard S-V fails maybe this would be useful for scaling $r$.

**Comment by j2schmit** [comment]
Good idea. This could work if we can come up with a good estimate of $\kappa$. The catch is that the official definition involves the notion of arc-length, which is actually a choice that's been discussed as a generic monitor function, but it doesn't have great computational efficiency, so we would need a cheap way to calculate that.

**Comment by j2schmit** [comment]
GIven we are currently using the hairer approach, we don't have to worry about taking a derivative of the monitor function anymore, so this could be a good choice. Can we estimate the radius of the osculating circle for arbitrary dynamics?
